tweets.sql

```diff
@@ -0,0 +1,36 @@
+-- USERDATA table for holding the user information (username, user full name, and user id)
+CREATE TABLE USERDATA(
+    userid NUMBER NOT NULL,
```

Comment:
Different than swagger naming convention (camelCase), in SQL language we usually follow the [naming convention](https://www.sqlstyle.guide/) that separates words by underscores (and usually all UPPERCASE). For example, `userid` should be `USER_ID`. Same with the table name.